pkg/scheduler/scheduler.go

```diff
@@ -285,15 +296,22 @@ func (s *Scheduler) schedule(ctx context.Context) {
 	}
 
 	// 6. Requeue the heads that were not scheduled.
+
+	// we backoff if there are no successful admissions
+	shouldBackoff := true
+
 	result := metrics.AdmissionResultInadmissible
 	for _, e := range entries {
 		logAdmissionAttemptIfVerbose(log, &e)
 		if e.status != assumed {
 			s.requeueAndUpdate(log, ctx, e)
 		} else {
 			result = metrics.AdmissionResultSuccess
+			shouldBackoff = false
 		}
 	}
+	s.handleBackoff(shouldBackoff)
```

Comment:
```suggestion
        // we backoff if there are no successful admissions
	s.handleBackoff(result != metrics.AdmissionResultSuccess)
```
non blocking nit, just to remove the redundant var

Reply:
thanks, done!